### ISSUE TITLE
Lint failing in CI

### ISSUE THREAD

**Issue by jobimrobinsantos-drizly** [hound-search/hound#419]
See for example, https://github.com/hound-search/hound/runs/6054555586
```
could not load export data: cannot import \"internal/goarch\" (unknown iexport format version 2), export data is newer version - update tool
```

My company's private fork of hound is failing in the same fashion. At a quick glance, it appears to be an issue of compatibility between go 1.13 (this repo) and 1.18 (what runs on github actions) or a problem with golangci-lint. For comparison, just 3 months ago this check was passing but github actions was running go 1.17 https://github.com/hound-search/hound/runs/4954299890

**Comment by salemhilal** [comment]
Thanks for opening an issue! It looks like we're [seeing the seeing the same issue](https://github.com/hound-search/hound/runs/6054555586?check_suite_focus=true) as well. We'll try and carve out some time to investigate. I wouldn't be surprised if the version compatibility was the culprit since not a ton else has changed. If you run into other clues, definitely let us know.

**Comment by salemhilal** [comment]
This is fixed as of #422! Thank you again for your help.

**Comment by jobimrobinsantos-drizly** [comment]
No, thank *you*!
